android/app/build.gradle

```diff
@@ -212,7 +212,7 @@ dependencies {
     implementation "com.android.support:appcompat-v7:26.1.0"
     // Force using exact RN version instead of relying on gradle dependency resolution
     // https://docs.gradle.org/current/userguide/introduction_dependency_management.html#sec:dependency_resolution
-    implementation ("com.facebook.react:react-native:0.53.3") { force = true }  // From node_modules
+    compile ("com.facebook.react:react-native:0.55.4") { force = true }  // From node_modules
```

Comment:
Don't we want to use `implementation` though? From original PR: https://github.com/status-im/status-react/pull/4446#discussion_r190916069